docs/configure-harvest-basic.md

```diff
@@ -329,7 +336,42 @@ password: mypassword
 EOF
 ```
 
-`get_credentials` that outputs only the password in plain text:
+`get_credentials` that outputs authToken in YAML format:
+```bash
+#!/bin/bash
+# script requests an access token from the authorization server
+# authorization returns an access token to the script
+# script writes the YAML formatted authToken like so:
+cat << EOF
+authToken: $authToken
+EOF
+```
+
+`get_credentials` that outputs authToken in YAML format for interacting with Keycloak auth provider:
```

Comment:
Below are a couple of examples of how to create a credentials script that uses an OAuth2 workflow to authenticate with ONTAP or StorageGRID OAuth-enabled clusters.

??? note "These are examples that you will need to adapt to your environment."

    Example OAuth2 script authenticating with the Keycloak auth provider via `curl`. Uses [jq](https://github.com/jqlang/jq) to extract the token. This script outputs the authToken in YAML format.
    
    ```bash
    #!/bin/bash

    response=$(curl "http://{KEYCLOAK_IP:PORT}/realms/{REALM_NAME}/protocol/openid-connect/token" \
      --header "Content-Type: application/x-www-form-urlencoded" \
      --data-urlencode "grant_type=password" \
      --data-urlencode "username={USERNAME}" \
      --data-urlencode "password={PASSWORD}" \
      --data-urlencode "client_id={CLIENT_ID}" \
      --data-urlencode "client_secret={CLIENT_SECRET}")

    access_token=$(echo "$response" | jq -r '.access_token')

    cat << EOF
    authToken: $access_token
    EOF
    ```

    Example OAuth2 script authenticating with the Auth0 auth provider via `curl`. Uses [jq](https://github.com/jqlang/jq) to extract the token. This script outputs the authToken in YAML format.

    ```bash
    #!/bin/bash
    response=$(curl https://{AUTH0_TENANT_URL}/oauth/token \
      --header 'content-type: application/json' \
      --data '{"client_id":"{CLIENT_ID}","client_secret":"{CLIENT_SECRET}","audience":"{ONTAP_CLUSTER_IP}","grant_type":"client_credentials"')
    
    access_token=$(echo "$response" | jq -r '.access_token')

    cat << EOF
    authToken: $access_token
    EOF
    ```

Reply:
Done